src/services/color/color.service.ts

```diff
@@ -91,7 +83,12 @@ export class ColorService {
     }
 
     getColor(color: ColorIdentifier): ThemeColor {
-        return this._colors[color];
+        return this._colors[color.toLowerCase()];
+    }
+
+    setTheme(theme: string[]) {
```

Comment:
It would be good to use get/set syntax here, so that the theme/color set can be read as well as assigned.